ui/summary.go

```diff
@@ -262,22 +274,29 @@ func SummarizeMetrics(w io.Writer, indent string, t time.Duration, timeUnit stri
 	extraMaxLens := make([]int, 2)
 
 	trendCols := make(map[string][]string)
-	trendColMaxLens := make([]int, len(TrendColumns))
+	trendColMaxLens := make([]int, len(s.trendColumns))
 
 	for name, m := range metrics {
 		names = append(names, name)
 
 		// When calculating widths for metrics, account for the indentation on submetrics.
-		displayName := DisplayNameForMetric(m) + IndentForMetric(m)
+		displayName := displayNameForMetric(m) + indentForMetric(m)
 		if l := StrWidth(displayName); l > nameLenMax {
 			nameLenMax = l
 		}
 
 		m.Sink.Calc()
 		if sink, ok := m.Sink.(*stats.TrendSink); ok {
-			cols := make([]string, len(TrendColumns))
-			for i, col := range TrendColumns {
-				value := m.HumanizeValue(col.Get(sink), timeUnit)
+			cols := make([]string, len(s.trendColumns))
+			for i, tc := range s.trendColumns {
+				resolver := s.trendValueResolvers[tc]
+				var value string
+				switch v := resolver(sink).(type) {
+				case float64:
+					value = m.HumanizeValue(v, timeUnit)
+				case uint64:
+					value = strconv.FormatUint(v, 10)
+				}
```

Comment:
This type switch and all resolvers returning `interface{}` is a drawback of this approach, but it was the cleanest way I found to handle the `count` column (resolves to `uint64`) in the same way as the others (which resolve to `float64`).

Reply:
This should be fine. We can remove it, if the value resolvers weren't returned 2 arguments (the second being the string representation of the value), or if instead of lambdas, they were objects with methods, one of which is `humanize() string` or something like that. In any case, not sure it's worth it for something so simple.